File: JS/PGS_124 나라의 숫자.js
function solution(n) {
    var answer = '';
    while(n > 0){
        a = parseInt(n / 3)
        b = n % 3
        if(b == 0){
            b = 4
            a -= 1
        }
        n = a
        answer = (b) + answer
    }
    return answer;
}

File: Python/PGS_124 나라의 숫자.py
def solution(n):    
    answer = ''
    n = int(n)
    while n > 0:
        a = n // 3
        b = n % 3
        if b == 0:
            b = 4
            a -= 1
        n = a
        answer = str(b)+answer
        
    return answer